StortSpelProjekt/Sandbox/src/main.cpp

```diff
@@ -114,12 +125,49 @@ int WINAPI WinMain(HINSTANCE hInstance, HINSTANCE, LPSTR, int nCmdShow)
             }
         }
 
+        static bool currentScene = true;
+        static int nameCounter = 0;
+        if (window->WasSpacePressed())
```

Comment:
In the future, these things are better to be put in an "update function" similar to the ones already existing. Might also look nicer to use the existing input or event system rather than circumventing it. Doesn't matter now, since this will be removed, just a tip for the next time

Reply:
True. but easier/faster for me to just do some static :). Will remove this when a second reviewer has commented some stuff.